app/models/bot/smooch.rb

```diff
@@ -782,7 +782,8 @@ def self.save_media_message(message)
       filepath = File.join(Rails.root, 'tmp', filename)
       media_type = "Uploaded#{message['type'].camelize}"
       File.atomic_write(filepath) { |file| file.write(data) }
-      pm = ProjectMedia.joins(:media).where('medias.type' => media_type, 'medias.file' => filename, 'project_medias.project_id' => message['project_id']).last
+      team_id = Project.find(message['project_id']).team_id
```

Comment:
It's better to use `find_by` instead of `find` to avoid `ActiveRecord::RecordNotFound` error
i.e `Project.find_by(id: message['project_id'])&.team_id`